style.css

```diff
@@ -0,0 +1,177 @@
+
+h1 {
+    font-size: 56px;
+    font-family: 'Segoe UI';
+    text-transform: uppercase;
+    text-align: center;
+    line-height: 1;
+}
+
+h2 {
+    font-size: 18px;
+    font-family: 'Segoe UI';
+}
+
+h3 {
+    font-size: 24px;
+    font-family: Tahoma;
+    text-transform: uppercase;
+    color: #ffffff;
+}
+
+.block-title {
+    background-color: rgb(245, 0, 102);
+}
+
+mark {
+    font-size: 20px;
+    font-family: 'Segoe UI';
+    color: #ffffff;
+    background-color: rgb(235, 64, 135);
+    text-transform: uppercase;
+    margin: 10px;
+    padding: 4px;
+}
+
+p {
+    font-family: 'Times New Roman';
+    font-size: 16px;
+    line-height: 2;
+}
+
+i {
+    font-family: serif;
+    font-size: 24px;
+    display: block;
+    text-align: center;
+    font-style: normal;
+    
+}
+
+.title-1 {
```

Comment:
Please think about more meaningful name. If you delete a title or change the order of titles name 'title-1' will be really confusing. I would call it 'story-by-title' if you want to call it title

![image](https://user-images.githubusercontent.com/104353194/169476725-367cad8d-6ad5-4976-b6b7-5b97aa768fe7.png)